calc/src/desc.ts

```diff
@@ -149,13 +149,15 @@ export function getRecovery(
       [minD, maxD] = multiDamageRange(damage) as [number[], number[]];
     }
     const percentHealed = move.drain[0] / move.drain[1];
-    const max = Math.round(defender.curHP() * percentHealed);
+    const attackerHasBigRoot = (attacker.hasItem('Big Root'));
+    let maxDrain = Math.round(defender.curHP() * percentHealed);
+    if (attackerHasBigRoot) maxDrain = Math.trunc(maxDrain * 5324 / 4096);
```

Comment:
To be honest, I'm not entirely sure whether this big root factor needs to take place before or after rounding; am going based on the calculation for `drained`